src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1491,6 +1491,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return typesXmlConverter;
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-linux`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Drawing.Primitives.dll; SideIndicator=<=}.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 126
126:         $diffs | Should -BeExactly $null
```
DeserializedMethods.Deserialized objects shouldn't ever have any methods (unless they are primitive known types)
```powershell
Cannot bind parameter 'MemberType'. Cannot convert value "*Method*" to type "System.Management.Automation.PSMemberTypes". Error: "*Method* is not a valid value for PSMemberTypes."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/DeserializedMethods.Tests.ps1: line 19
19:         $d | Get-Member -MemberType *Method* Add | Should -BeNullOrEmpty

```